lstests/core/service.py

```diff
@@ -0,0 +1,1517 @@
+                    w.show_lineno = helper.override(DEFAULT_SHOW_LINENO, w.show_lineno, self.cla.show_lineno)
+                    w.show_function = helper.override(DEFAULT_SHOW_FUNCTION, w.show_function, self.cla.show_function)
+                    w.show_thread = helper.override(DEFAULT_SHOW_THREAD, w.show_thread, self.cla.show_thread)
+                    w.show_log_level = helper.override(DEFAULT_SHOW_LOG_LEVEL, w.show_log_level, self.cla.show_log_level)
+            # empty log files
+            elif isinstance(w, FileWriter):
+                if not self.cla.dont_rename and self.cla.cluster_id is not None:
+                    bas, ext = os.path.splitext(w.filename)
+                    w.filename = f"{bas}-{self.cla.cluster_id}{ext}"
+                with open(w.filename, "w"):
+                    pass
+        # register for shorter names
+        self.log = self.logger.log
+        self.trace = self.logger.trace
+        self.debug = self.logger.debug
+        self.info = self.logger.info
+        self.warn = self.logger.warn
+        self.error = self.logger.error
+        self.fatal = self.logger.fatal
+        self.flag = self.logger.flag
+        self.flush = self.logger.flush
+
+    def connect(self):
+        self.info(">>> [Connect to Service] ---------------- BEGIN ----------------------------------------------------")
+        self.print_working_dir()
+        self.print_system_info()
+        self.print_config()
+        if self.addr == "127.0.0.1":
+            self.connect_to_local_service()
+        else:
+            self.connect_to_remote_service()
+        self.info(">>> [Connect to Service] ---------------- END ------------------------------------------------------")
+
+    def print_working_dir(self):
+        self.print_header("working directory")
+        self.debug("{:22}{}".format("Working Directory", os.getcwd()))
+
+    def print_system_info(self):
+        self.print_header("system info")
+        self.debug("{:22}{}".format("UTC Time", time.strftime("%Y-%m-%d %H:%M:%S %Z", time.gmtime())))
+        self.debug("{:22}{}".format("Local Time", time.strftime("%Y-%m-%d %H:%M:%S %Z", time.localtime())))
+        self.debug("{:22}{}".format("Platform", platform.platform()))
+
+    def print_config(self):
+        self.print_header("service configuration")
+        # print service config
+        self.print_config_helper("-a: addr",  HELP_ADDR,  self.addr,  DEFAULT_ADDR)
+        self.print_config_helper("-o: port",  HELP_PORT,  self.port,  DEFAULT_PORT)
+        self.print_config_helper("-w: wdir",  HELP_WDIR,  self.wdir,  DEFAULT_WDIR)
+        self.print_config_helper("-f: file",  HELP_FILE,  self.file,  DEFAULT_FILE)
+        self.print_config_helper("-g: gene",  HELP_GENE,  self.gene,  DEFAULT_GENE)
+        self.print_config_helper("-s: start", HELP_START, self.start, DEFAULT_START)
+        self.print_config_helper("-k: kill",  HELP_KILL,  self.kill,  DEFAULT_KILL)
+        # print stdout logger config
```

Comment:
Thanks for the comment. One logger keeps logging config to multiple destinations, including, usually, the `stdout`, and multiple log files. To avoid dumping too much information, I chose to print out only the config related to `stdout` here. So I guess maybe we don't want to remove the "stdout" from description.